### PR TITLE
Remove revision announcer completely

### LINKED ISSUE #1738: Turn down revision announcer
Based on Cloud Monitoring, there is 0 request to this service in the past 6 months. Users have switched to the epoch branches of the wpt repo (maintained with [GitHub Actions](https://github.com/web-platform-tests/wpt/blob/master/.github/workflows/epochs.yml)). I think the time has come for us to turn this service down. Here is the plan

1. [x] Disable traffic serving of announcer
2. [x] Delete the announcer service from AppEngine, including its routing settings
3. [ ] Delete related code in this repo

I propose a somewhat arbitrary 1 week buffer time between each step.

cc @stephenmcgruer @foolip

### PR DESCRIPTION
**Pull request by Hexcles**
Fixes #1738 .

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -275,7 +271,7 @@ gcloud-%: gcloud
 node-%: node
 	@ echo "# Installing $*..."
 	# Hack to (more quickly) detect whether a package is already installed (available in node).
-	cd webapp; node -p "require('$*/package.json').version" 2>/dev/null || npm install --no-save $*
+	cd webapp; node -p "require('$*epackage.json').version" 2>/dev/null || npm install --no-save $*
```
Comment (stephenmcgruer) [inline Makefile]: What is this change about?

File: docs/app-engine.md [hunk 2]
```diff
@@ -7,7 +7,8 @@ each of which has an `app.yaml` file in its directory.
    APIs.
 2. **processor**: `/results-processor/`, the internal backend of the Results
    Receiver (not accessible externally) which processes the incoming results.
-3. **announcer**: `/revisions/service/`, the Revision Announcer.
+3. **searchcache**: `/api/query/cache/service/`, an in-memory cache and query
+   executor for [structured search](../api/query/README.md).
```
Comment (stephenmcgruer) [inline docs/app-engine.md]:
```suggestion
   executor for [structured searches](../api/query/README.md).
```

Maybe?

File: Makefile (hunk 1 above)
Comment (Hexcles) [reply Makefile]: That must be some devil living in my vim :)

```suggestion
	cd webapp; node -p "require('$*/package.json').version" 2>/dev/null || npm install --no-save $*
```

Comment (Hexcles) [reply Makefile]: That must be some devil living in my vim :)

```suggestion
	cd webapp; node -p "require('$*/package.json').version" 2>/dev/null || npm install --no-save $*
```

**Pull request merged**